### DISCUSSION TITLE
Option to watch/follow a post

### DISCUSSION THREAD

**Discussion by naim94a** [bluesky-social/atproto#2514] [Bluesky Client App]
If I'm interested in a discussion, it would be useful to have a way to subscribe to the thread and receive notifications for new replies

**Comment by bnewbold** [comment]
Yup, this is a frequent request. It didn't make [our recent product roadmap](https://bsky.social/about/blog/05-07-2024-product-roadmap), but it is on our list and we'll probably get to it before too long.
